### PR TITLE
Add `communicate-on-pull-request-merged` GitHub Action

### PR DESCRIPTION
**Pull request by mollyIV**
### Description

The action will:

* add a comment (default `.. Thank you for your contribution to ...`) to a pull request that is merged
* add a label (default `status: included-in-next-release`) to a pull request that is merged 

Please note that this is just an action implementation. It does not take any effect itself. A new workflow which uses the action will be created. 

### Testing 

To test the changes, run unit tests by executing `npm test` (followed by `npm install` to install the package dependencies).

```bash
 PASS  __tests__/main.test.ts
  action test suite
    ✓ It posts a comment on a merged issue for (pull-request-closed.json) (1406ms)
    ✓ It does not post a comment on a closed pull request for (issue.json) (14ms)
    ✓ It does not post a comment on a closed pull request for (action-opened.json) (13ms)
    ✓ It does not post a comment on a closed pull request for (pull-request-closed-but-not-merged) (9ms)

Test Suites: 1 passed, 1 total
Tests:       4 passed, 4 total
Snapshots:   0 total
Time:        2.182s, estimated 3s
Ran all test suites.
```

I also tested the workflow on my private repo (comments from my personal account, because I set myself as a bot 😅):

image

🎊 

If you want to test this action, here is the workflow to use:

```yaml
name: Processing pull requests
on: 
  pull_request:
    types: [closed]

jobs:   
  pr-release-lifecycle:
    name: PR release lifecycle
    runs-on: ubuntu-latest    
    steps:
    - uses: actions/checkout@master
      with:
        ref: refs/heads/master
    - uses: mollyIV/github-actions/communicate-on-pull-request-merged@pr-release-lifecycle
      with:
        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

closes https://github.com/fastlane/github-actions/issues/12.

### CODE REVIEW

File: communicate-on-pull-request-merged/.prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "printWidth": 80,
+    "tabWidth": 2,
+    "useTabs": false,
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "none",
+    "bracketSpacing": false,
+    "arrowParens": "avoid",
+    "parser": "typescript"
+  }
```
Comment (janpio) [inline communicate-on-pull-request-merged/.prettierrc.json]: formatting

File: communicate-on-pull-request-merged/LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+The MIT License (MIT)
+
+Copyright (c) 2018 The Fastlane Authors
```
Comment (janpio) [inline communicate-on-pull-request-merged/LICENSE]: 2019 :)

File: communicate-on-pull-request-merged/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Communicate on pull request merged
+
+An action for adding comments and labels to a pull request which is merged.
```
Comment (janpio) [inline communicate-on-pull-request-merged/README.md]: "... pull request when it is merged."

File: communicate-on-pull-request-merged/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+```yaml
+steps:
+- uses: fastlane/github-action/communicate-on-pull-request-merged@latest
+  with:
+    repo-token: ${{ secrets.GITHUB_TOKEN }}
+```
+
+# License
+
+The scripts and documentation in this project are released under the [MIT License](LICENSE)
```
Comment (janpio) [inline communicate-on-pull-request-merged/README.md]: Licenes could/should probably be a top level thing, not in each action.

**Comment by mollyIV** [comment]
Oh, it's been merged already, don't you mind Jan if I open another pull request and apply your suggestions there?

**Comment by janpio** [comment]
Oh, that overlapped.

Yes of course, consider all of them and create a new PR if useful/necessary.

**Comment by mollyIV** [comment]
The PR is ready for review: https://github.com/fastlane/github-actions/pull/16 @janpio 😊 🙇

**Pull request merged**